Source/driver/Castro_setup.cpp

```diff
@@ -259,6 +259,33 @@ Castro::variableSetUp ()
   small_dens = new_min_rho;
   EOSData::mindens = new_min_rho;
 
+  // Given small_temp and small_dens, compute small_pres
+  // and small_ener, assuming a more restrictive value is
+  // not already provided by the user. We'll arbitrarily
+  // set the mass fraction for this call, since we presumably
+  // don't need to be too accurate, we just need to set a
+  // reasonable floor.
+
+  eos_t eos_state;
+
+  eos_state.rho = castro::small_dens;
+  eos_state.T = castro::small_temp;
+  for (int n = 0; n < NumSpec; ++n) {
+      eos_state.xn[n] = 1.0_rt / NumSpec;
+  }
+#if NAUX_NET > 0
+  for (int n = 0; n < NumAux; ++n) {
+      // This value is arbitrary, we may want to fix this
+      // later if it causes a problem.
+      eos_state.aux[n] = 1.0_rt;
+  }
```

Comment:
I think if we are doing NSE_THERMO, we should properly call `set_nse_aux_from_X(state_t& state)`

Reply:
OK. I did this and removed what was there since it wasn't great, we can find a better solution later if we have non-NSE problems using aux.